ciao-controller/main.go

```diff
@@ -136,6 +136,8 @@ func main() {
 	identityURL = clusterConfig.Configure.IdentityService.URL
 	serviceUser = clusterConfig.Configure.Controller.IdentityUser
 	servicePassword = clusterConfig.Configure.Controller.IdentityPassword
+	keyringPath = clusterConfig.Configure.Storage.SecretPath
```

Comment:
Yes, those 2 fields should be required through validMinConf() but you want to allow for a grace period and give people time to update their cluster configuration, i.e.:

* `validMinConf()` should spit a warning for now if the storage config is not present for e.g. a couple of weeks. Then after that we make it a hard requirement.
* Send an email to the ciao mailing list about the upcoming requirement for the storage configuration entry.
* keyringPath should be set to the default value if we receive a "" string from the configuration server. And we should keep it like that for the long term.